libs/native/jna/src/main/java/org/elasticsearch/nativeaccess/jna/JnaNativeLibraryProvider.java

```diff
@@ -45,6 +51,11 @@ public JnaNativeLibraryProvider() {
         );
     }
 
+    @SuppressForbidden(reason = "jna library path must be set for load library to work with our own libs")
+    private static void setJnaLibraryPath() {
+        System.setProperty("jna.library.path", LoaderHelper.platformLibDir.toString());
```

Comment:
Will `toString` return a path with the right separators to be parsed properly on all platforms?

Reply:
Yes. The javadoc for `Path.toString()` mentions the following:
> The returned path string uses the default name separator to separate names in the path.

Reply:
Yes. The javadoc for `Path.toString()` mentions the following:

> The returned path string uses the default name separator to separate names in the path.

Reply:
Sounds good. I wasn't totally certain the "default name separator" was one that worked on all platforms.